gulpfile.mjs

```diff
@@ -2226,7 +2226,7 @@ function packageJson() {
     },
     repository: {
       type: "git",
-      url: `git+${DIST_REPO_URL}.git`,
+      url: `git+https://github.com/mozilla/pdf.js.git`,
```

Comment:
Nit: Maybe we should define a `REPO_URL` variable, placed with the other definitions at the top the `packageJson` function, and use that one here?

Reply:
We can get rid of `DIST_REPO_URL` altogether because it's only used to initialize an empty Git repository, but that is no longer necessary since https://github.com/mozilla/pdf.js/pull/18350/commits/a5f2b9faebe7b9f315a572916d9a80fa9c05471b removed the Git logic it was used for. I just forgot to remove it in that commit, and for now decided to do that after this release to get that out of the door first and to not tempt fate any further ;-)